bundle/edu.gemini.spModel.io/src/main/java/edu/gemini/spModel/io/impl/PioSpXmlParser.java

```diff
@@ -231,6 +232,9 @@ private ISPRootNode _parseDocument(Document doc) throws Exception {
         // Update pre-2017A programs
         To2017A.updateProgram(doc);
 
+        // Update pre-2017B programs
+        To2018A.updateProgram(doc);
```

Comment:
I guess you mean pre-2018A in the comment.  There is a 2017B migration too, but I guess that will be revealed when you merge.